docs/components/connectors/out-of-the-box-connectors/embeddings-vector-db.md

```diff
@@ -111,20 +116,98 @@ Camunda document reference metadata, similarity score, and the actual text conte
 
 ## Embedding models
 
-### Amazon Bedrock
+<Tabs groupId="embedding-models" defaultValue="bedrock" queryString values={
+[
+{label: 'Amazon Bedrock', value: 'bedrock' },
+{label: 'Azure OpenAI', value: 'azure-openai' },
+{label: 'Google Vertex AI', value: 'vertex-ai' },
+{label: 'OpenAI', value: 'openai' }
+]}>
 
+<TabItem value='bedrock'>
 The **vector database connector** currently supports only [Amazon Titan V1/V2 models](https://docs.aws.amazon.com/bedrock/latest/userguide/titan-embedding-models.html).
```

Comment:
Should we remove the `only` here?

In general, I think it would be great if we could align the Bedrock documentation structure with the other 3 providers by describing the parameters.

Reply:
I agree :+1: